modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -3770,8 +3772,6 @@ private static boolean waitForTopology(TestCluster cluster, PeerId peerId, int e
         RaftGroupService grp = cluster.getServer(peerId);
 
         if (grp == null) {
-            log.warn("Node has not been found {}", peerId);
```

Comment:
Why is this removed?

Reply:
Because it accesses the `log` field, which is no longer static. Making this method not static is not trivial, however, and I think it's easier to remove this not very useful message

Reply:
You could just add another static logger field to this class and use it

Reply:
makes sense